### DISCUSSION TITLE
wooooo!

### DISCUSSION THREAD

**Discussion by oldtun** [share121/inter-knot#4619] [灌水]
这就是真正的绳网，真正的新艾利都。来到末日最高城，太美丽了新艾利都

**Comment by shidijun** [comment]
看的出来是萌新绳匠 初入绳网吧<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >
